docs/ref/models/fields.txt

```diff
@@ -1157,6 +1157,35 @@ you can use the name of the model, rather than the model object itself::
         # ...
         pass
 
+Note that relationships defined this way on
+:ref:`abstract models <abstract-base-classes>` are only resolved when the
+model is subclassed as concrete model and are not relative to the abstract
+model's ``app_label``::
+
+    from django.db import models
+
+    class AbstractCar(models.Model):
+        manufacturer = models.ForeignKey(
+            'Manufacturer',
+            on_delete=models.CASCADE,
+        )
+
+        class Meta:
+            app_label = 'products'
```

Comment:
maybe using the "snippet" directive with a filename would be better than declaring an explicit `app_label`?